python/mleap/__init__.py

```diff
@@ -0,0 +1,3 @@
+from mleap.version import version
+
+__version__ = version
```

Comment:
@lucagiovagnoli let me know if you know a better way to do this.

Reply:
I think the convention is to have this `__init__.py` file contain:
```
__version__ = 0.16.0-SNAPSHOT
```
rather than importing it from another version.py file.

So just move the contents of version.py in this file :)

Reply:
I tried that, but because it's used in setup.py, it didn't work to put it here, because i think when setup.py runs, this doesn't exist yet.